glue/flagging_script_glue/flagging_e894e5.py

```diff
@@ -1085,22 +1095,14 @@ def name_selector(tokens) -> str:
         'Empty Name' if name is empty.
         id (str): identified last name
     """
-    if not tokens:
+
+    if tokens == "Empty Name" or tokens == []:
         return "Empty Name"
-    if tokens == "Empty Name":
-        return tokens
-    # Ex: John Smith Jr
+
     if tokens[-1] in ["jr", "sr", "ii", "iii", "iv", "v"]:
         tokens = tokens[:-1]
```

Comment:
**nitpick:** What if the last two elements match one of these? i.e. James Kirk Jr. III.